### PR TITLE
Send notice instead of hard fail

### PR DESCRIPTION
**Pull request by propyless**
thought it might be better to send a notice rather than fail the whole puppet run.
this is just to see if you think this might be better.

//propy

### CODE REVIEW

File: manifests/limits/fragment.pp [hunk 1]
```diff
@@ -10,36 +10,36 @@
   include pam
   include pam::limits
   if $::osfamily == 'Suse' and $::lsbmajdistrelease == '10' {
-    fail('You can not use pam::limits::fragment together with Suse 10.x releases')
```
Comment (ghoneycutt) [inline manifests/limits/fragment.pp]: defines/classes should be explicit about what they support and if you use an unsupported system, they should fail.

**Comment by propyless** [comment]
Alright yeah thats what I thought as well. If it the OS does not support pam fragments, the puppet run should fail imo. We had some mixed opinions about it failing here at work and I thought i'd provide an alternate solution.

**Comment by ghoneycutt** [comment]
Thanks for asking. Always better to fail the catalog when unsure of compatibility as opposed to doing something that may not work as intended. If you checkout out the pam class you'll see the osfamily case statement where the defaults are always to fail.

**Comment by propyless** [comment]
There opinion derives from the fact that people (including us) have been using fragments and sle 10.x and now we would receive fails.

**Comment by Phil-Friderici** [comment]
thanks for your thoughts :)

**Comment by propyless** [comment]
closed because it should fail.

**Pull request closed without merging**